### ISSUE TITLE
CI is broken

### ISSUE THREAD

**Issue by phil-davis** [owncloud/web#4549]
https://drone.owncloud.com/owncloud/web/12398/4/6
```
+ cp tests/drone/config.json dist/config.json
+ mkdir -p /srv/config
+ cp -r /var/www/owncloud/web/tests/drone /srv/config
BusyBox v1.31.1 () multi-call binary.

Usage: cp [OPTIONS] SOURCE... DEST

Copy SOURCE(s) to DEST

	-a	Same as -dpR
	-R,-r	Recurse
	-d,-P	Preserve symlinks (default if -R)
	-L	Follow all symlinks
	-H	Follow symlinks on command line
	-p	Preserve file attributes if possible
	-f	Overwrite
	-i	Prompt before overwrite
	-l,-s	Create (sym)links
	-T	Treat DEST as a normal file
	-u	Copy only newer files
```

**Comment by phil-davis** [comment]
That pipeline step is using https://hub.docker.com/r/webhippie/nodejs/tags?page=1&ordering=last_updated

`latest` was updated 6 months ago, so there is not any recent change to that docker image that might have broken something.

**Comment by phil-davis** [comment]
Fixed by #4550
